Exebite.API/Controllers/CommentsController.cs

```diff
@@ -51,6 +51,14 @@ public class CommentsController : ControllerBase
                    .Reduce(_ => NotFound(), error => error is RecordNotFound, x => _logger.LogError(x.ToString()))
                    .Reduce(_ => InternalServerError(), x => _logger.LogError(x.ToString()));
 
+        [HttpDelete("{id}")]
+        [Authorize(Policy = nameof(AccessPolicy.DeleteMealAccessPolicy))]
+        public IActionResult Delete(int id, [FromBody] DeleteCommentDto comment) =>
```

Comment:
Usually the `{id}` route property is enough to find the record that should be deleted. Since our `CommentEntity` has a composite primary key, the delete endpoint could have looked like following:

```suggestion
        [HttpDelete("{commentId}/meal/{mealId}")]
        [Authorize(Policy = nameof(AccessPolicy.DeleteMealAccessPolicy))]
        public IActionResult Delete(int commentId, long mealId) =>
```